nvidia.go

```diff
@@ -0,0 +1,116 @@
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"golang.org/x/sys/unix"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+const ldconfPath = "/etc/ld.so.conf"
+
+func setupNvidiaDriver(containerRootfs string) (err error) {
+	bins, libs, err := getNvidiaFiles()
+	if err != nil {
+		return
+	}
+
+	// mount driver binaries
+	for _, binFile := range bins {
+		if err = mount(binFile, filepath.Join(containerRootfs, binFile), "bind", unix.MS_BIND|unix.MS_REC, "rbind"); err != nil {
```

Comment:
just to make sure the binaries can't be modified (in case a user adds `rw` in the kernel command line), could you add `unix.MS_RDONLY` and check if this still works for you?

Reply:
yeah，it works. 
To add.